src/Aggregate/aggregate.py

```
@@ -0,0 +1,421 @@
+'''
+Description:
+These are general functions to be used in the river node, raster, 
+and lake processors for aggregating heights and areas with their 
+corresponding uncertainties
+
+
+Copyright (c) 2018-, California Institute of Technology ("Caltech"). U.S.
+Government sponsorship acknowledged.
+All rights reserved.
+
+Author (s): Brent Williams
+'''
+
+import numpy as np
+
+def simple(in_var, metric='mean'):
```

Do not use nan versions of stat methods, they hide errors upstream

---

fixed in dacb948fa40ae5c17346861d74c83e3267bf84c4